cylc/flow/xtrigger_mgr.py

```diff
@@ -186,6 +194,32 @@ class XtriggerManager:
     managed uniquely - i.e. many tasks depending on the same clock trigger
     (with same offset from cycle point) get satisfied by the same call.
 
+    Parentless tasks with xtrigger(s) are, by default, spawned out to the
+    runahead limit. This results in non-sequential, and potentially
+    unnecessary, checking out to this limit (and may introduce clutter to
+    user interfaces). An option to make this sequential is now available,
+    by changing the default for all xtriggers in a workflow, and a way to
+    override this default with a (reserved) keyword function argument
+    (i.e. "sequential=True/False"):
+
+    # Example:
+    [scheduling]
+        sequential xtrigger default = True
+        [[xtriggers]]
+            clock_0 = wall_clock()  # offset PT0H
+            clock_1 = wall_clock(offset=PT1H)
+                 # or wall_clock(PT1H)
+            workflow_x = workflow_state(
+                workflow=other,
+                point=%(task_cycle_point)s,
+                sequential=False
+            ):PT30S
+        [[graph]]
+            PT1H = '''
+                @clock_1 & @workflow_x => foo & bar
+                @wall_clock = baz  # pre-defined zero-offset clock
+            '''
```

Comment:
This example is near-identical to the one above. Just delete it and add `sequential=False` to the one above?

Reply:
And this config item doesn't exist (anymore):
`   sequential xtrigger default = True`

Reply:
The purpose was to show how you could modify the above example to make the spawning behave differently.. Actually now that `wall_clock` is sequential this doesn't quite achieve that ..

Reply:
Have modified it somewhat to emphasize the ways this option can modify the spawning behavior of a similar workflow.